pkg/log/multislogger/multislogger.go

```diff
@@ -8,6 +8,25 @@ import (
 	slogmulti "github.com/samber/slog-multi"
 )
 
+type contextKey int
+
+func (c contextKey) String() string {
```

Comment:
Do we need these strings at all?

Reply:
yea, the middle ware needs to know what to set the key of the attribute to in the log